src/components/Navbar.jsx

```diff
@@ -0,0 +1,30 @@
+import { PropTypes } from 'prop-types';
+import { AiOutlineLeft, AiTwotoneSetting } from 'react-icons/ai';
+import { BsMicFill } from 'react-icons/bs';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = ({ view, year }) => (
+  <div className="range">
+    <div>
+      <NavLink to="/">
+        <AiOutlineLeft />
```

Comment:
### [OPTIONAL]
You may hide this on the home page because it can confuse the user for going back further. 👍 
<img width="318" alt="Screenshot 2023-06-30 at 1 47 59 AM" src="https://github.com/francksefu/capstone-react/assets/83048208/9ef56940-ea59-4204-8919-d2187656fda3">

Reply:
This change has been done well